Updates CSPM / KSPM Findings page (#4532)
elastic/security-docs: docs/cloud-native-security/cspm-findings.asciidoc

By default, the Findings page lists all findings, without grouping or filtering.

[discrete]
=== Group findings 

. Click **Group findings by:** to view a list of fields by which you can group findings. Select one of the suggested group-by fields, or click **Custom field** to choose your own group-by field. 

> . When grouping is enabled, click a group to expand it and view all findings within that group.

Reviewer: Optional edit to language to avoid using ableist terms.

```suggestion
. When grouping is turned on, click a group to expand it and examine all findings within that group.
```

> . To disable grouping, click **Group findings by:** and select **None**.

Reviewer: Same same 

```suggestion
. To turn off grouping, click **Group findings by:** and select **None**.
```

=== Customize the Findings table
Use the toolbar buttons in the upper-left of the Findings table to customize the columns you want displayed:

* **Columns**: Select the left-to-right order in which columns appear.
* **Sort fields** or **_n_ fields sorted**: Sort the table by one or more columns, or turn sorting off.

> * **Fields**: Select which fields to display for each finding. Fields which are checked here appear in the table and the **Columns** menu.

Reviewer: ```suggestion
* **Fields**: Select which fields to display for each finding. Selected fields appear in the table and the **Columns** menu.
```

> TIP: You can also click a column's name to open a menu that allows you to remove the column, sort by the column, move the column left or right, copy the column's name, or copy its contents as a string.

Reviewer: Optional edit to future-proof this list of actions. 

```suggestion
TIP: You can also click a column's name to open a menu that allows you to perform multiple actions on the column. 
```
